tensorflow_model_optimization/g3doc/guide/clustering/clustering_example.ipynb

```diff
@@ -0,0 +1,515 @@
+{
```

Comment:
<p>Thinking this through the</p><p>" You can run this jupyter notebook in your local&nbsp;<a href="https://www.tensorflow.org/install/pip?lang=python3#2.-create-a-virtual-environment-recommended" target="_blank">virtualenv</a>&nbsp;or colab. For details of setting up dependencies, please refer to&nbsp;<a href="https://app.reviewnb.com/tensorflow/model-optimization/pull/330/files/" target="_blank">comprehensive guide</a>." is better suited in general installation instructions. </p><p> </p><p>If you can create a PR for virtualenv in <a href="https://www.tensorflow.org/model_optimization/guide/install" target="_blank">https://www.tensorflow.org/model_optimization/guide/install</a>, that'd be great. </p><p> </p><p>A comment above ! pip install tensorflow ... that points to <a href="https://www.tensorflow.org/model_optimization/guide/install" target="_blank">https://www.tensorflow.org/model_optimization/guide/install</a> would be fine. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Another guy from our team will raise the PR for this and I left the TODO in the notebook to remind myself to add the link later.</p>